### PR TITLE
feat(ebpf): make ptrace no rely on sys_enter/exit

### PR DESCRIPTION
**Pull request by OriGlassman**
### 1. Explain what the PR does
make ptrace no rely on sys_enter/exit

"Replace me with `make check-pr` output"

### 2. Explain how to test it
./tracee -e=ptrace

### CODE REVIEW

File: pkg/ebpf/c/tracee.bpf.c [hunk 1]
```diff
@@ -5133,6 +5133,31 @@ int BPF_KPROBE(trace_security_task_setrlimit)
     return events_perf_submit(&p, 0);
 }
 
+SEC("kprobe/ptrace")
```
Comment (NDStrahilevitz) [inline pkg/ebpf/c/tracee.bpf.c:5136]: Not sure if there is a raw tracepoint for this, but i'm pretty sure that even a regular tracepoint has better performance than a kprobe.

Comment (NDStrahilevitz) [reply pkg/ebpf/c/tracee.bpf.c:5136]: According to @yanivagman in #2085, where I suggested something similar with a regular tracepoint, there should be an option for a raw tracepoint.

Comment (OriGlassman) [reply pkg/ebpf/c/tracee.bpf.c:5136]: If you succeed at doing a raw_tracepoint on a syscall that's indeed preferred. Let me know if that's the case.
tracepoint requires debugfs which is not currently a requirement of tracee.

**Comment by yanivagman** [comment]
Replaced by #4256

**Pull request closed without merging**